### DISCUSSION TITLE
How to fix Error 404

### DISCUSSION THREAD

**Discussion by RyanneBaumann** [community/community#123613] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

It was all working fine on my website, and suddenly, without me changing anything, this error appeared:
<img width="500" alt="Bildschirmfoto 2024-05-13 um 09 30 33" src="https://github.com/community/community/assets/142094035/6d4e028a-b812-43f7-9a94-251d2f66eddb">

The error only occurs with one HTML file. I've tried to figure out what's causing it or how to fix it, but I haven't made much progress.

**Comment by davevad93** [comment]
Hi @RyanneBaumann ,

In case you haven't checked it out yet, take a look at this:

- [Troubleshooting 404 errors for GitHub Pages sites](https://docs.github.com/en/pages/getting-started-with-github-pages/troubleshooting-404-errors-for-github-pages-sites)
